INSTALL.md

```diff
@@ -72,3 +72,13 @@ Gentoo
 * x11-misc/xdg-utils
 * gnome-extra/zenity
 * net-misc/curl
+
+
+OSX
+=====
+
+Use `make dependencies` to map the native libraries to your system.
+
+To compile OpenRA, run `make all` from the command line.
```

Comment:
Thanks. Just confirmed it works on my system. I've added a commit to reflect this.